packages/nodes-base/nodes/Wise/GenericFunctions.ts

```diff
@@ -65,6 +70,95 @@ export async function wiseApiRequest(
 	}
 }
 
+/**
+ * Make an authenticated SCA API request to Wise.
+ */
+ export async function wiseSCAApiRequest(
+	this: IHookFunctions | IExecuteFunctions | ILoadOptionsFunctions,
+	method: 'GET' | 'POST' | 'PUT' | 'DELETE' | 'HEAD',
+	endpoint: string,
+	body: IDataObject = {},
+	qs: IDataObject = {},
+	option: IDataObject = {},
+) {
+	const { apiToken, environment, privateKey } = await this.getCredentials('wiseApi') as {
+		apiToken: string,
+		environment: 'live' | 'test',
+		privateKey: string | null,
+	};
+
+	const rootUrl = environment === 'live'
+		? 'https://api.transferwise.com/'
+		: 'https://api.sandbox.transferwise.tech/';
+
+	const options: IHttpRequestOptions = {
+		headers: {
+			'user-agent': 'n8n',
+			'Authorization': `Bearer ${apiToken}`,
+		},
+		method,
+		url: `${rootUrl}${endpoint}`,
+		qs,
+		body,
+		json: true,
+		returnFullResponse: true,
+		validateStatus: (status) => {
+			return (status >= 200 && status < 300) || status === 403; // default
+		},
+	};
+
+	if (!Object.keys(body).length) {
+		delete options.body;
+	}
+
+	if (!Object.keys(qs).length) {
+		delete options.qs;
+	}
+
+	if (Object.keys(option)) {
+		Object.assign(options, option);
+	}
+
+	let response;
+	try {
+		response = await this.helpers.httpRequest!(options);
+	} catch (error) {
+		throw new NodeApiError(this.getNode(), error);
+	}
+
+	if (response.statusCode === 200) {
+		return response.body;
```

Comment:
Note to myself to check when testing. The original calls `this.helpers.request` and returns the response unaltered. This version calls `this.helpers.httpRequest` and returns `response.body`. Same at L153. Does this change preserve the shape of the original response?

Reply:
As above, the `request` helper just returns `response.body`